examples/topography_spectra.jl

```diff
@@ -0,0 +1,133 @@
+using LazyArtifacts
+using ClimaAtmos.AtmosArtifacts
+import ClimaAtmos as CA
+import ClimaCore as CC
+import ClimaComms
+import ClimaParams as CP
+import ClimaCore.Quadratures
+using ClimaUtilities: SpaceVaryingInputs
+using ClimaCore.Hypsography: diffuse_surface_elevation!
+using ClimaCore: Remapping, Geometry, Operators, Spaces
+using Plots, ClimaCorePlots
+
+import ClimaCoreSpectra: power_spectrum_2d
+
+const AA = AtmosArtifacts
+
+"""
+    generate_spaces(; h_elem=60, diffiter=32)
+For a given number of elements `h_elem`, this function generates a 
+horizontal spectral space (ClimaCore.SpectralElementSpace2D) and
+generates an interpolated (using SpaceVaryingInputs) orography field
+on this space. The `diffuse_surface_elevation` function is called 
+to apply `diffiter` instances of Laplacian diffusion passes 
+(which rely on the canonical unsteady diffusion equation).
+Returns a `Field` containing surface orography. 
+"""
+function generate_spaces(; h_elem = 60, diffiter = 32)
+    FT = Float32
+    param_dict = CP.create_toml_dict(FT)
+    params = CP.get_parameter_values(param_dict, ["planet_radius"])
+    cubed_sphere_mesh =
+        CA.cubed_sphere_mesh(; radius = params.planet_radius, h_elem)
```

Comment:
```suggestion
function generate_spaces(; h_elem = 60, diffiter = 32, planet_radius = 6.378e6)
    FT = Float32
    cubed_sphere_mesh =
        CA.cubed_sphere_mesh(; radius = FT(planet_radius), h_elem)
```
I think it is an overkill to have this function depending on `ClimaParams` when all we need is an Earth radius. This is also just an example.

Reply:
Sure, this was intended as a quick demo of `params` usage in `CA` (I realise that the `ClimaParams` documentation also has this. I'll change this to just depend on the keyword arg.